packages/lab/src/stepped-tracker/TrackerConnector/TrackerConnector.css

```diff
@@ -0,0 +1,22 @@
+.saltTrackerConnector {
+  --trackerConnector-style-default: var(--saltTrackerConnector-style-default, var(--salt-track-borderStyle-incomplete));
+  --trackerConnector-style-active: var(--saltTrackerConnector-style-active, var(--salt-track-borderStyle-active));
+  --trackerConnector-color: var(--saltTrackerConnector-color, var(--salt-track-borderColor));
+  --trackerConnector-thickness: var(--saltTrackerConnector-thickness, var(--salt-track-borderWidth));
+  --trackerConnector-margin: var(--saltTrackerConnector-margin, var(--salt-spacing-50));
+}
```

Comment:
don't think you need to use these vars, they're not changing their values so you can just use the value directly

Reply:
I added these as they give consumers a way to override certain aspects of the design without needing a complex selector. Personally I think we should do this more, but we should have a general consensus on it.